clang/lib/Lex/PPDirectives.cpp

```diff
@@ -2096,7 +2096,9 @@ Preprocessor::ImportAction Preprocessor::HandleHeaderIncludeOrImport(
   // Issue a diagnostic if the name of the file on disk has a different case
   // than the one we're about to open.
   const bool CheckIncludePathPortability =
-      !IsMapped && !File->getFileEntry().tryGetRealPathName().empty();
+//      !IsMapped && !File->getFileEntry().tryGetRealPathName().empty();
+// Disable file portability check to unblock pulldown
+      false;
```

Comment:
This looks like an issue which is not related to the SYCL changes. Do you see the same problems in llvm CI?

Reply:
This is reproduced only on windows with Unix-like console.
I am going to disable check until CI will move to pure windows environment

Reply:
> This is reproduced only on windows with Unix-like console.
> I am going to disable check until CI will move to pure windows environment

This is useful information, which should be added in the comments rather than "to unblock pulldown".
Please, make sure that this issue is tracked and resolved.

Reply:
I have created https://github.com/intel/llvm/issues/1594 to track that.